src/main/scala/ai/privado/tagger/config/RubyDBConfigTagger.scala

```diff
@@ -0,0 +1,83 @@
+package ai.privado.tagger.config
+
+import ai.privado.cache.DatabaseDetailsCache
+import ai.privado.languageEngine.java.language.NodeStarters
+import ai.privado.model.DatabaseDetails
+import ai.privado.tagger.{PrivadoParallelCpgPass, PrivadoSimpleCpgPass}
+import ai.privado.utility.Utilities.addDatabaseDetailsMultiple
+import io.shiftleft.codepropertygraph.generated.Cpg
+import io.shiftleft.codepropertygraph.generated.nodes.JavaProperty
+import org.slf4j.LoggerFactory
+import overflowdb.traversal._
+
+class RubyDBConfigTagger(cpg: Cpg) extends PrivadoSimpleCpgPass(cpg) {
```

Comment:
Can we not have a `BaseeDBConfigTagger` which have functions defined for parsing different database files like postgresDb dynamo db. And in the language specific tagger, we see any different implementation just override them?

Reply:
Also this is a ruby tagger so it should be in language Engine ruby's package

Reply:
Yes Please, we need to start modelling using the OOPS concept. I see we are writing a lot of duplicate code

Reply:
done